docs/source/quark_script.rst

```diff
@@ -271,6 +271,18 @@ activityInstance.isExported(none)
 - **params**: none
 - **return**: True/False
 
+getApplication(samplePath)
+==========================
+- **Description**: Get the application from the manifest of the target sample.
+- **params**: 
+    1. ``samplePath`` : the file path of the target sample
+- **return**: Application instance of the sample
+
+applicationInstance.isDebuggable(none)
+==========================
```

Comment:
According to [the reStructuredText spec.](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections), the underline should be as long as the title.